src/components/Proposal/Create/PluginForms/CreateUnknownGenericPluginProposal.tsx

```diff
@@ -109,193 +138,274 @@ class CreateGenericPlugin extends React.Component<IProps, IStateProps> {
-          initialValues={this.currentFormValues}
-          // eslint-disable-next-line react/jsx-no-bind
-          validate={(values: IFormValues): void => {
-            const errors: any = {};
-
-            this.currentFormValues = values;
-
-            const require = (name: string) => {
-              if (!(values as any)[name]) {
-                errors[name] = "Required";
-              }
-            };
-
-            const nonEmpty = (name: string) => {
-              if (!(values as any)[name].toString()) {
-                errors[name] = "Required";
-              }
-            };
-
-            const nonNegative = (name: string) => {
-              if ((values as any)[name] < 0) {
-                errors[name] = "Please enter a non-negative value";
-              }
-            };
-
-            if (values.title.length > 120) {
-              errors.title = "Title is too long (max 120 characters)";
-            }
-
-            if (!isValidUrl(values.url)) {
-              errors.url = "Invalid URL";
-            }
-
-            const bytesPattern = new RegExp("0x[0-9a-f]+", "i");
-            if (values.callData && !bytesPattern.test(values.callData)) {
-              errors.callData = "Invalid encoded function call data";
-            }
-
-            require("callData");
-            require("title");
-            require("description");
-            require("value");
-            nonEmpty("value");
-            nonNegative("value");
-
-            return errors;
-          }}
-          onSubmit={this.handleSubmit}
-          // eslint-disable-next-line react/jsx-no-bind
-          render={({
-            errors,
-            touched,
-            // eslint-disable-next-line @typescript-eslint/no-unused-vars
-            handleSubmit,
-            isSubmitting,
-            resetForm,
-            // eslint-disable-next-line @typescript-eslint/no-unused-vars
-            setFieldTouched,
-            setFieldValue,
-          }: FormikProps<IFormValues>) =>
-            <Form noValidate>
-              <TrainingTooltip overlay={i18next.t("Title Tooltip")} placement="right">
-                <label htmlFor="titleInput">
-                  <div className={css.requiredMarker}>*</div>
-                Title
-                  <ErrorMessage name="title">{(msg: string) => <span className={css.errorMessage}>{msg}</span>}</ErrorMessage>
-                </label>
-              </TrainingTooltip>
-              <Field
-                autoFocus
-                id="titleInput"
-                maxLength={120}
-                placeholder={i18next.t("Title Placeholder")}
-                name="title"
-                type="text"
-                className={touched.title && errors.title ? css.error : null}
-              />
+  private getEncodedData = (abi: Array<any>, name: string, values: any[]) => {
+    const encodedData = encodeABI(abi, name, values);
+    this.setState({ callData: encodedData });
```

Comment:
check if `encodedData` ==""

Reply:
Why?

Reply:
because encodeABI return "" if the Abi is not valid

Reply:
In case when `encodeABI` returns "" it means at least one of the parameters are not valid / not supplied. So that's why it shows nothing in the encoded data.
Maybe instead of showing nothing I can give a message inside the encodedData box like: "please fill the params to generate the encoded data".

Reply:
ok